pci/src/bus.rs

```diff
@@ -23,21 +24,28 @@ const DEVICE_ID_INTEL_VIRT_PCIE_HOST: u16 = 0x0d57;
 const NUM_DEVICE_IDS: usize = 32;
 
 /// Errors for device manager.
-#[derive(Debug)]
+#[derive(Error, Debug)]
 pub enum PciRootError {
     /// Could not allocate device address space for the device.
+    #[error("Could not allocate device address space for the device")]
```

Comment:
I think the convention is that we put the embedded error after the message with a colon separating them - like `"Could not allocate device address space for the device: {0}"`
```
rockhopper:~/src/cloud-hypervisor (main)$ git grep "#\[error.*: {0}" | wc -l
586
```

Reply:
In which case we don't need to add that here.

Reply:
So for this commit, I follow the `": {0}"` convention for the inner error for all enum variants that I touch anyway. This is likely going to be removed again in #7066, but this is another story